mechanical/assembly_renderer.py

```diff
@@ -23,7 +23,7 @@
 from nimble_build_system.cad.shelf import create_shelf_for
 
 assembly_definition_file = "../build/assembly-def.yaml"
-
+render_destination = os.path.join(os.getcwd(), "assembly-docs", "renders")
```

Comment:
Actually thinking about it. `assembly-docs` is automatically created by GitBuilding when it runs. So this is the one location we can't use or it gets overwritten by GitBuilding. If we change it to just "renders" than this can be accessed from the documentation documentation dir as  `../build/renders`.

```suggestion
render_destination = os.path.join(os.getcwd(), "renders")
```